config/config.local.js

```diff
@@ -0,0 +1,4 @@
+export const config = {
+    mail: "mail",
```

Comment:
May be copy values from prod config? Number and email are environment-independent